src/genie/libs/parser/iosxe/tests/test_show_ip_nbar_version.py

```diff
@@ -0,0 +1,75 @@
+# Python 
+import unittest
+from unittest.mock import Mock
+
+# ATS
+from ats.topology import Device
+from ats.topology import loader
+
+# Metaparser
+from genie.metaparser.util.exceptions import SchemaEmptyParserError, SchemaMissingKeyError
+
+from genie.libs.parser.iosxe.show_ip_nbar_version import ShowIpNbarVersion
+
+# ==============================
+# Unittest for:
+#    * 'show ip nbar version'
+# ==============================
+class test_show_ip_nbar_version(unittest.TestCase):
+    '''
+    Unit test for:
+    show ip nbar version
+    '''
+
+    device = Device(name = 'aDevice')
+    empty_output = { 'execute.return_value': '' }
+
+    #* define the golden_parsed_output(s) and golden_output(s) here
+    golden_parsed_output_1 = {
```

Comment:
sorry, not sure. but based on the output `Loaded Protocol Pack(s)`(plural), I assume it could be possible to have multiple items like below.
```
Loaded Protocol Pack(s): 


Name:                            Advanced Protocol Pack
Version:                         42.0
Publisher:                       Cisco Systems Inc.
NBAR Engine Version:             34
Creation time:                   Thu Mar 28 10:40:34 UTC 2019
File:                            bootflash:sdavc/pp-adv-all-169.1-34-42.0.0.pack
State:                           Inactive


Name:                            Advanced Protocol Pack
Version:                         41.0
Publisher:                       Cisco Systems Inc.
NBAR Engine Version:             31
Creation time:                   Mon Feb 11 09:42:11 UTC 2019
File:                            bootflash:sdavc/pp-adv-all-166.2-31-41.0.0.pack
State:                           Active
```
so, please change the schema to below:
```
    schema = {
        'nbar_software_version': str,
        'nbar_minimum_backward_compatible_version': str,
        'loaded_protocol_packs': {
            Any(): { # name
                'version': {
                    Any(): {
                        'file': str,
                        'publisher': str,
                        'creation_time': str,
                        'nbar_engine_version': str,
                        'state': str,
                    },
                },
            },
        }
    }

Reply:
Fixed!